test/Microsoft.Health.SqlServer.Tests.Integration/Features/Schema/SchemaUpgradeRunnerTests.cs

```diff
@@ -62,9 +62,11 @@ public async Task ApplySchema_BaseSchemaExists_Succeeds()
         }
 
         [Fact]
-        public async Task ApplySchema_UsingDiff_Succeeds()
+        public async Task ApplySchemaOnPreviouslyFailedAttempt_UsingDiff_Succeeds()
```

Comment:
Should not we add another test and keep the existing one because they are two different usecases?